.github/workflows/ci.yml

```diff
@@ -0,0 +1,201 @@
+name: C++ CI Workflow
+
+# template derived from https://github.com/robotology/how-to-export-cpp-library/tree/master/.github/workflows 
+# and https://github.com/robotology/idyntree/blob/master/.github/workflows/ci.yml
+
+on:
+    push:
+    pull_request:
+    schedule:
+    # run a cron job for a nightly build
+    # * is a special character in YAML so you have to quote this string
+    # Execute a "nightly" build at 2 AM UTC
+    - cron: '0 2 * * *'
+    
+env:
+    vcpkg_robotology_TAG: v0.0.3
+    YCM_TAG: v0.11.1
+    YARP_TAG: v3.3.2
+    iDynTree_TAG: v1.1.0   
+    # Overwrite the VCPKG_INSTALLATION_ROOT env variable defined by Github Actions to point to our vcpkg
+    VCPKG_INSTALLATION_ROOT: C:\robotology\vcpkg
```

Comment:
I think we can try to refresh a bit the version used for CI, it probably make sense to try to use the latest stable release for these dependencies. For the vcpkg_robotology, note that for new versions since 0.1.0 the snippet of code to use them in GitHub Actions changed: https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/releases/tag/v0.1.0 .

Reply:
Thanks @traversaro for noting it, I have updated the dependencies in https://github.com/robotology/wearables/pull/94/commits/09aac0a1754800309105cdd449bc178c5daaeacb.